src/main/scala/org/broadinstitute/dsp/HelmInterpreter.scala

```diff
@@ -3,65 +3,79 @@ package org.broadinstitute.dsp
 import cats.data.Kleisli
 import cats.effect.concurrent.Semaphore
 import cats.effect.{Async, Blocker, ContextShift}
-import cats.implicits._
 import com.sun.jna.Native
 import io.chrisdavenport.log4cats.Logger
 import org.broadinstitute.dsp.implicits._
 import scala.language.implicitConversions
 
-class Helm[F[_]: ContextShift](blocker: Blocker,
-                               concurrencyBound: Semaphore[F])(implicit logger: Logger[F], F: Async[F]) {
-  val helmClient = Native.load(
-    "helm",
-    classOf[HelmJnaClient])
+class Helm[F[_]: ContextShift](blocker: Blocker, concurrencyBound: Semaphore[F])(implicit logger: Logger[F],
```

Comment:
Should this be

```suggestion
final class HelmInterpreter[F[_]: ContextShift](blocker: Blocker, concurrencyBound: Semaphore[F])(implicit logger: Logger[F],
```

Reply:
Yeah that's our convention I think, can update..